src/ai/backend/logging/utils.py

```diff
@@ -56,3 +64,14 @@ def enforce_debug_logging(loggers: Iterable[str]) -> None:
     for name in loggers:
         instance = logging.getLogger(name)
         instance.setLevel(logging.DEBUG)
+
+
+@contextmanager
+def with_log_context_fields(fields: Mapping[str, Any]) -> Generator[Mapping[str, Any], None, None]:
```

Comment:
Doesn't we need `current_log_context_fields`?

Reply:
The situation is designed such that log context fields cannot be accessed externally. They are only read from BraceStyleAdapter.